config/locales/en.yml

```diff
@@ -64,7 +64,7 @@ en:
     error:
       generic: |
         Oops! Something went wrong and we couldn't process that action. You
-        might want to try it again, or if that doesn't work, refresh the page.
+        might want to try it again.
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8000202/24160949/7dbfabe6-0e39-11e7-9eb0-928e0d67b53f.png)

I have attached the image just so we have some context. Would it make sense to attach a reload button to the message?